999-SAPonAzure/Student/03-SAP-Security.md

```diff
@@ -0,0 +1,45 @@
+# Challenge 3: Secure SAP on Azure
+
+[< Previous Challenge](./02-Azure-Monitor.md) - **[Home](../README.md)** - [Next Challenge >](./04-BusinessContinuity-and-DR.md)
+
+## Introduction
+
+Contoso Inc. has deployed the SAP systems, activated Azure monitoring for SAP; now the team is looking to setup security checks and threat monitoring for the SAP application and its respective infrastucture components.
+
+## Description
+
+In this challenge we will be enrolling the Azure subscirption for Security Center and Azure defender, and also configure Setinel workspace for active SAP application security monitoring. This will give us an opportunity to learn how to use the Security Center Dashboard, check for security KPI of Azure resources and also enable continous SAP application-tier threat monitoring.
+
+#### This challenge has two parts
+1) Security center and Azure defender (intro and basics).
+
+2) Azure Sentinel (requires Sentinel workspace and SAP knowledge) Detailed documentation provided seperately.
+
+## Tips
+- If you are an Internal Microsoft employee and built SAP system using provided terraform script, note below mentioned points before proceeding to part 2 of this challenge.
```

Comment:
There should be NO mention of special instructions for "internal Microsoft employees" in the student guide!  This is a public facing repo and that could be exposure of private company IP.

Are these "Tips" relevant to all students?  If so, keep them, but describe them as so.  If not, then I would remove these Tips.

Reply:
I see the update. Thanks for this!   When you say "if the SAP system was deployed with terraform in C1"...  what is the other possibility?   Meaning if it were pre-provisioned?